usaspending_api/search/v2/es_sanitization.py

```diff
@@ -0,0 +1,14 @@
+import logging
```

Comment:
If `es_sanitize` is being segregated, would it be agreeable to move both it and `es_minimal_sanitize` together in the `usaspending-api/usaspending_api/common/elasticsearch/` directory?

Reply:
that's very agreeable